css-pseudo-4/Overview.bs

```diff
@@ -696,11 +696,12 @@ Styling Highlights</h3>
 
   Issue: Are there any other properties that should be included here?
 
-  The 'color' property specifies the color of both the text
-  and all line decorations (underline, overline, line-through)
-  and emphasis marks ('text-emphasis')
-  applied to the text
-  by the <a>originating element</a> and its ancestors and descendants.
+  'color' and 'background-color'
+  are known as <dfn>highlight colors</dfn>.
+  Because this concept exists for compatibility reasons,
+  other similar properties
+  like 'fill-color' or 'stroke-color'
+  must not be treated as highlight colors.
```

Comment:
it looks like i moved this definition up in 65597ae6af59211f1c857c6509cf696bcdbf52df. at the time it linked to the section explaining the impl impact, and i wanted to use the term “highlight colors” in the “Historically” note, but neither of those are true now, so i’ll move it back.

Reply:
done in 6a142341ace64adff79c1f622025630fb131b67a. i also eliminated the term “highlight colors” completely, which existed in the current draft but doesn’t really add any value as a separate term to “paired default highlight colors”, since the set of properties will never change (and we’re even adding a note to clarify this).